apps/web-giddh/src/app/ledger/ledger.component.ts

```diff
@@ -1392,15 +1393,19 @@ export class LedgerComponent implements OnInit, OnDestroy {
     }
 
     public showUpdateLedgerModal(txn: ITransactionItem, type: string) {
-        let transactions: TransactionsResponse = null;
-        this.store.pipe(select(t => t?.ledger?.transactionsResponse), take(1)).subscribe(trx => transactions = trx);
-        if (transactions) {
-            this.store.dispatch(this.ledgerActions.setAccountForEdit(this.lc.accountUnq));
+        if (txn?.adjustmentEntry) {
+            this.router.navigate([`/pages/inventory/v2/product/adjust/${txn?.description}`]);
```

Comment:
Check requirement basis

Reply:
As confirmed with Rishi and Aashish this key is correct. if any question then talks to them.